client/src/components/RecipeList/RecipeList.js

```diff
@@ -13,21 +14,22 @@ const ListContainer = styled.div`
 `;
 
 const RecipeList = () => {
+  const [recipes, setRecipes] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      const getRecipes = await getRecipies();
+      setRecipes(getRecipes);
+    }
+    fetchData();
+    console.log(recipes + " hallo");
```

Comment:
`console.log` is fine for checking if something works, shouldn't be merged in the end.

```suggestion

```